pallets/msa/src/tests/msa_token_tests.rs

```diff
@@ -0,0 +1,105 @@
+use frame_support::{
+	assert_err, assert_ok,
+	traits::{
+		tokens::{fungible::Inspect, Fortitude, Preservation},
+		Currency,
+	},
+};
+
+use sp_core::{sr25519, Pair};
+
+use crate::{tests::mock::*, Config, Error};
+
+use common_primitives::{
+	msa::H160,
+	signatures::{AccountAddressMapper, EthereumAddressMapper},
+};
+
+use pallet_balances::Event as BalancesEvent;
+
+#[test]
+fn it_succeeds_when_balance_is_sufficient() {
```

Comment:
I suggest adding another another test when the balance is 0.

Reply:
I tried; you can't set the balance to zero unless you do it in the genesis, and even then I'm not sure the account wouldn't get reaped before you could execute test method. Didn't seem worth it to go to all that effort.